packages/closer/pages/admin/config.tsx

```
@@ -359,231 +361,233 @@ const ConfigPage = ({ defaultEmailsConfig, error }: Props) => {
+                                          <option value={option} key={option}>
+                                            {option}
+                                          </option>
+                                        );
+                                      })}
+                                    </select>
+                                  )}
+                                </div>
+                              )}
+                            </div>
+                          )}
+                        </>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)